.github/workflows/validate-datadog-changes.yml

```diff
@@ -0,0 +1,32 @@
+name: Validate changes to DataDog Service Catalog Files
+on:
+ pull_request:
+   paths:
+     - "datadog-service-catalog/**.yml"
+
+jobs:
+  validate_files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: setup python env
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+          cache: 'pip'
+      - name: Install python requirements
+        working-directory: ./.github/scripts/validate-yaml
+        run: pip install -r requirements.txt
+      - name: Get Filenames
+        id: get_filenames
+        run: |
+              git fetch origin master --depth 1
+              echo changedFiles=$(git diff --diff-filter=d --name-only origin/master... -- "datadog-service-catalog/**.yml") >> $GITHUB_OUTPUT
+      - name: validate files
```

Comment:
```suggestion
      - name: Validate Service Catalog Entries
```

Reply:
I made it generic because the script will validate any yaml against a json schema. I can rename it if you think that is better